data/scenarios/Challenges/Ranching/beekeeping.yaml

```diff
@@ -0,0 +1,453 @@
+version: 1
+name: Beekeeping
+author: Karl Ostmo
+description: |
+  Bootstrap an organic *bee*verage industry.
+creative: false
+seed: 2
+attrs:
+  - name: bee
+    fg: '#ffff00'
+    bg: '#000000'
+  - name: water_cask
+    fg: '#4488ff'
+    bg: '#8B4513'
+  - name: mybase
+    fg: '#e0c0e0'
+  - name: iceblue
+    fg: '#ddddff'
+objectives:
+  - teaser: Apiarist
+    goal:
+      - |
+        Build a `beehive`{=structure}.
+      - |
+        This will attract bees, which will gather
+        `nectar`{=entity} from nearby `wildflower`{=entity}s.
+      - |
+        Do not crowd hives; provide at least one cell margin
+        between them.
+      - |
+        Given that `wildflower`{=entity}s cannot be relocated,
+        judicious placement is essential for efficient
+        `honeycomb`{=entity} production.
+    condition: |
+      foundStructure <- structure "beehive" 0;
+      return $ case foundStructure (\_. false) (\_. true);
+  - teaser: Collect honeycomb
+    goal:
+      - |
+        Collect `honeycomb`{=entity} from the `beehive`{=structure}s.
+      - |
+        After gathering a certain amount of nectar, a bee will return
+        to its hive and place honeycomb alongside it.
+        A finite amount of honeycomb may be accumulated before the
+        bee becomes dormant.
+    condition: |
+      as base {
+        has "honeycomb"
+      }
+  - teaser: Cooper
+    goal:
+      - |
+        Make a `cask`{=entity}.
+      - |
+        Utilize local timber and your on-hand supply of iron material.
```

Comment:
This is just a silly pet peeve, but I hate the word "utilize".  There's nothing wrong with the good old-fashioned word "use".